APIs_Drivers/BufferedSerial_printf/main.cpp

```diff
@@ -0,0 +1,20 @@
+#include "mbed.h"
+
+// Create a new UARTSerial object with a default baud rate.
+static BufferedSerial serial_port(USBTX, USBRX, 9600);
```

Comment:
The only usage of this is if an application wants to use a different UART other than the console UART.
So perhaps we should write 
```
static BufferedSerial serial_port(TARGET_TX_PIN, TARGET_RX_PIN, 9600);
```

Reply:
[Done](https://github.com/ARMmbed/mbed-os-examples-docs_only/compare/dd81220d2f1c723333609afa51c1479d12251021..17b7244223e0208efdc12274754ea22467bc8234).